database_import.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+
+sleep 30
+
+DATABASE_TO_COPY_INTO="baseball"
+DATABASE_FILE="baseball.sql"
+PASS="ROOT_ACCESS_PASSWORD"
+
+mysql -u root -p${PASS} -h mariadb -e "CREATE DATABASE IF NOT EXISTS ${DATABASE_TO_COPY_INTO}"
+mysql -u root -p${PASS} -h mariadb ${DATABASE_TO_COPY_INTO} < ${DATABASE_FILE}
+echo "DATABASE IMPORT DONE....."
+echo "Starting HW......."
+
+mysql -u root -p${PASS} -h mariadb baseball < src/SQL_BAIN_OF_MY_EXISTENCE.sql > results.csv
```

Comment:
I do not see your results.csv file...maybe you want to check on that.

Reply:
Hi Meghan, once you run the container and receive the message "Done....", you have to make a new terminal and type the command 
`docker exec -it bda_602_SQL_ROL_AVG_1 /bin/bash` 
This should take you to the home directory of the container and there you can find results.csv